src/ISComm.c

```diff
@@ -1152,6 +1152,20 @@ char copyDataBufPToStructP(void *sptr, const p_data_buf_t *data, const unsigned
     }
 }
 
+char copyDataPToDataP(p_data_t *dst, const p_data_t *src, const unsigned int maxsize)
```

Comment:
Needs a doxygen.  Also, doesn't this function exist elsewhere?  Or has it been moved here?